src/coverlet.console/Program.cs

```diff
@@ -6,24 +6,26 @@
 using System.Text;
 
 using ConsoleTables;
-using coverlet.console;
 using Coverlet.Console.Logging;
 using Coverlet.Core;
 using Coverlet.Core.Abstracts;
 using coverlet.core.Enums;
 using Coverlet.Core.Enums;
 using Coverlet.Core.Extensions;
+using Coverlet.Core.Helpers;
 using Coverlet.Core.Reporters;
 using McMaster.Extensions.CommandLineUtils;
+using Microsoft.Extensions.DependencyInjection;
 
 namespace Coverlet.Console
 {
     class Program
     {
         static int Main(string[] args)
         {
-            var logger = Services.Current.GetService<ILogger>();
-            var fileSystem = Services.Current.GetService<IFileSystem>();
+            var serviceProvider = GetServiceProvider();
```

Comment:
Build container here and avoid to pass to `DependencyInjectionExtensions` we'll remove it from core.

Reply:
Done and removed ```DependencyInjectionExtensions```